api/swagger/server/restapi/configure_thyra_server.go

```diff
@@ -21,6 +22,17 @@ func configureFlags(api *operations.ThyraServerAPI) {
 }
 
 func configureAPI(api *operations.ThyraServerAPI) http.Handler {
+	// unused
+	return nil
+}
+
+func (s *Server) ConfigureMassaStationAPI(config config.AppConfig) {
+	if s.api != nil {
+		s.handler = configureMassaStationAPI(s.api, config)
+	}
```

Comment:
should we handle the else case here? what happen if server has no api? is it even possible?

Reply:
i dont know but its copy pasta from swagger generated code.. i guess for the else case thyra will not start